.github/workflows/main.yml

```diff
@@ -51,7 +51,7 @@ jobs:
 
       - name: Running Tests
         run: |
-          python -m pytest --run-network-tests --verbose --cov=virtualizarr --cov-report=xml
+          python -m pytest --run-network-tests --verbose --cov-report=xml
```

Comment:
This fixes a warning `CoverageWarning: --include is ignored because --source is set (include-ignored)
    self.warn("--include is ignored because --source is set", slug="include-ignored")`